### DISCUSSION TITLE
`Payload error: JSON payload (34692 bytes) is larger than allowed (limit: 32768 bytes).`

### DISCUSSION THREAD

**Discussion by dxphilo** [actix/actix-web#3457] [General]
I am sending both form data and some json data to an endpoint that is powered by actix-web. Unfortnately I get the error `Payload error: JSON payload (34692 bytes) is larger than allowed (limit: 32768 bytes).`.

What I have tried.

I have tried setting the configuration of the form,json and payloads (10MB) as shown below but for some reason I still get the error.

My config:
```rs
        let json_cfg = web::JsonConfig::default()
            .limit(10 * 1024 * 1024) // 10 MB limit
            .error_handler(|err, _req| {
                error::InternalError::from_response(err, HttpResponse::Conflict().into()).into()
            });

        let multipart_cfg = web::FormConfig::default().limit(10 * 1024 * 1024);
        let payload_cfg = web::PayloadConfig::default().limit(10 * 1024 * 1024);

        App::new()
            .app_data(app_data.clone())
            .app_data(json_cfg.clone())
            .app_data(multipart_cfg.clone())
            .app_data(payload_cfg.clone())

```

On my client I can view the data on the network tab as shown below. Not sure what I might be missing.

<img width="502" alt="Screenshot 2024-08-20 at 17 12 58" src="https://github.com/user-attachments/assets/9c6822db-22e7-48fb-9765-2a60234c2b28">

**Comment by robjtede** [comment]
Are you using the `MultipartForm` derive for this? Can you show a bit more of the handler code please?

**Reply by dxphilo** [reply]
Mmhhh. I have reviewed this and it seems I was doing something completely wrong in my handler. 

In my case I was appending the json to the formData field and in my handler I was calling it directly with json: web::json<MyStruct>. My bad on this, i'll have to close this.
